scripts/ai/controllers/CutterController.lua

```diff
@@ -0,0 +1,34 @@
+--- Raises/lowers the pickup.
+---@class CutterController : ImplementController
+CutterController = CpObject(ImplementController)
+
+function CutterController:init(vehicle, implement)
+    ImplementController.init(self, vehicle, implement)
+    self.cutterSpec = self.implement.spec_cutter
+end
+
+function CutterController:onLowering()
+	self.implement:aiImplementStartLine()
```

Comment:
why is this called here again? Didn't we call this for all implements already from the strategy?